### PR TITLE
[FLINK-28998] Translate'Fine-Grained Resource Management' page into Chinese

### PR DESCRIPTION
**Pull request by liujingmao**
## Brief change log

- Translate "Fine-Grained Resource Management" page into Chinese:https://nightlies.apache.org/flink/flink-docs-master/zh/docs/deployment/finegrained_resource/
- The markdown file location: flink/docs/content.zh/docs/deployment/finegrained_resource.md, most of them have been translated, but some part has not been translated into Chinese.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): ( no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no )
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? ( no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: docs/content.zh/docs/deployment/finegrained_resource.md [hunk 1]
```diff
@@ -23,97 +23,86 @@ specific language governing permissions and limitations
...
-For users who wish to fine-tune their resource consumption, based on knowledge of their specific scenarios, Flink offers **fine-grained resource management**.
+# 细粒度资源管理
 
-This page describes the fine-grained resource management’s usage, applicable scenarios, and how it works.
+Apache Flink 努力为所有开箱即用的应用程序自动派生合理的默认资源需求。对于希望更精细化调节资源消耗的用户，基于对特定场景的了解，Flink 提供了**细粒度资源管理**。
+本文介绍了细粒度资源管理的使用、适用场景以及工作原理。
 
 {{< hint warning >}}
-**Note:** This feature is currently an MVP (“minimum viable product”) feature and only available to [DataStream API]({{< ref "docs/dev/datastream/overview" >}}).
+**Note:** 本特性是当前的一个最简化产品(版本)的特性，它支持只在DataStream API [DataStream API]({{< ref "docs/dev/datastream/overview" >}})中使用。
```
Comment (liuzhuang2017) [inline docs/content.zh/docs/deployment/finegrained_resource.md:33]: "Note" 最好翻译为 "注意"，"它支持只在DataStream API" 这里最好是中文和英文之间有空格，最好为 “它支持只在  
 DataStream API”

**Review by liuzhuang2017 — CHANGES_REQUESTED** [review]

File: docs/content.zh/docs/deployment/finegrained_resource.md (hunk 1 above)
Comment (liujingmao) [reply docs/content.zh/docs/deployment/finegrained_resource.md]: Thank you for your suggestion, has modified

File: docs/content.zh/docs/deployment/finegrained_resource.md [hunk 2]
```diff
@@ -23,97 +23,86 @@ specific language governing permissions and limitations
...
+Flink之前的资源申请只包含必须指定的slots,但没有精细化的资源配置,这是一种粗粒度的资源管理.在这种管理方式下, TaskManager以固定相同的slots的个数的方式来满足资源需求。
 {{< /hint >}}
 
-For the resource requirement without a specified resource profile, Flink will automatically decide a resource profile.
-Currently, the resource profile of it is calculated from [TaskManager’s total resource]({{< ref "docs/deployment/memory/mem_setup_tm" >}})
-and [taskmanager.numberOfTaskSlots]({{< ref "docs/deployment/config" >}}#taskmanager-numberoftaskslots), just
-like in coarse-grained resource management. As shown above, the total resource of TaskManager is 1 Core and 4 GB memory and the number of task slots
-is set to 2, *Slot 2* is created with 0.5 Core and 2 GB memory for the requirement without a specified resource profile.
+对于没有指定资源配置的资源请求，Flink会自动决定资源配置。粗粒度资源管理当前被计算的资源来自TaskManager总资源[TaskManager’s total resource]({{< ref "docs/deployment/memory/mem_setup_tm" >}})和TaskManager的总slot数[taskmanager.numberOfTaskSlots]({{< ref "docs/deployment/config" >}}#taskmanager-numberoftaskslots)。
+如上所示，TaskManager的总资源是1Core和4G内存，task的slot数设置为2，*Slot 2* 被创建，并申请0.5core和2G的内存而没有指定资源配置。
```
Comment (liuzhuang2017) [inline docs/content.zh/docs/deployment/finegrained_resource.md:63]: “TaskManager的总资源是1Core和4G内存，task的slot数设置为2，*Slot 2* 被创建，并申请0.5core和2G的内存而没有指定资源配置。” _这里最好翻译为_ “ TaskManager 的总资源是 1 Core 和  4 GB 内存，task 的 slot 数设置为 2，*Slot 2* 被创建，并申请 0.5 Core 和 2 GB 的内存而没有指定资源配置。”

File: docs/content.zh/docs/deployment/finegrained_resource.md [hunk 3]
```diff
@@ -23,97 +23,86 @@ specific language governing permissions and limitations
...
+在分配slot1和slot2后，在TaskManager留下0.25核和1G的内存作为未使用资源.
```
Comment (liuzhuang2017) [inline docs/content.zh/docs/deployment/finegrained_resource.md:64]: 在分配 *Slot 1* 和 *Slot 2* 后，在 TaskManager 留下 0.25 Core 和 1 GB 的内存作为未使用资源.

File: docs/content.zh/docs/deployment/finegrained_resource.md [hunk 4]
```diff
@@ -23,97 +23,86 @@ specific language governing permissions and limitations
...
 
-After the allocation of *Slot 1* and *Slot 2*, there is 0.25 Core and 1 GB memory remaining as the free resources in the
-TaskManager. These free resources can be further partitioned to fulfill the following resource requirements.
+详情请参考资源分配策略 [Resource Allocation Strategy](#resource-allocation-strategy)。
```
Comment (liuzhuang2017) [inline docs/content.zh/docs/deployment/finegrained_resource.md:66]: "详情请参考资源分配策略 [Resource Allocation Strategy](#resource-allocation-strategy)。" _这里最好翻译为_  “详情请参考 [资源分配策略](#resource-allocation-strategy)。”

File: docs/content.zh/docs/deployment/finegrained_resource.md [hunk 5]
```diff
@@ -23,97 +23,86 @@ specific language governing permissions and limitations
...
-  - Configure to enable fine-grained resource management.
+- 配置细粒度的资源管理
 
-  - Specify the resource requirement.
+- 指定资源请求
 
-### Enable Fine-Grained Resource Management
-
-To enable fine-grained resource management, you need to configure the [cluster.fine-grained-resource-management.enabled]({{< ref "docs/deployment/config" >}}#cluster-fine-grained-resource-management-enabled) to true.
+### Enable 细粒度资源管理
```
Comment (liuzhuang2017) [inline docs/content.zh/docs/deployment/finegrained_resource.md:77]: "Enable 细粒度资源管理" _最好翻译为_ “启用细粒度资源管理” 后面的 “enable” 也统一翻译为 “启用”

**Review by liuzhuang2017 — CHANGES_REQUESTED** [review]

File: docs/content.zh/docs/deployment/finegrained_resource.md [hunk 6]
```diff
@@ -232,93 +226,68 @@ ssg_with_resource = SlotSharingGroup.builder('ssg') \
...
-### Resource Allocation Strategy
-
-In this section, we talk about the slot partitioning mechanism in Flink runtime and the resource allocation strategy, including how
-the Flink runtime selects a TaskManager to cut slots and allocates TaskManagers on [Native Kubernetes]({{< ref "docs/deployment/resource-providers/native_kubernetes" >}})
-and [YARN]({{< ref "docs/deployment/resource-providers/yarn" >}}). Note that the resource allocation strategy is pluggable in
-Flink runtime and here we introduce its default implementation in the first step of fine-grained resource
-management. In the future, there might be various strategies that users can select for different scenarios.
+### 资源分配策略
 
+本节讨论的是Flink作业运行时的Slot分区机制和资源分配策略,包括在 YARN 和 Kubernetes 中运行 Flink 作业时,Flink 如何选择 TaskManager 来切分成 Slots 和如何分配 TaskManager 的.({{< ref "docs/deployment/resource-providers/native_kubernetes" >}})
```
Comment (liuzhuang2017) [inline docs/content.zh/docs/deployment/finegrained_resource.md:278]: 这里缺少了 [Native Kubernetes]({{< ref "docs/deployment/resource-providers/native_kubernetes" >}})，还有 Flink Slot 前后需要空格。

**Review by liuzhuang2017 — CHANGES_REQUESTED** [review]

**Review by liuzhuang2017 — APPROVED** [review]
LGTM

**Comment by liuzhuang2017** [comment]
@MartijnVisser , Sorry to bother you, can you help` @liujingmao ` review this pull request? grateful.

**Pull request merged**